### ISSUE TITLE
ES6 module export - js formatter adds erroneous linebreak

### ISSUE THREAD

**Issue by billrawlinson** [akalongman/sublimetext-codeformatter#148]
Consider the following code:

```javascript
'use strict';

class SampleClass {
 ...
}

export default SampleClass;
```

After executing the javascript formatter the code is changed so that the last line is split into two after the word export:

```javascript
export
default SampleClass;
```

It should not split that line.

**Comment by akalongman** [comment]
Fixed in 2.6.1 version

**Comment by billrawlinson** [comment]
great; thanks for taking care of this!
